surveys/forms.py

```diff
@@ -1,21 +1,57 @@
 from django import forms
-from pldp.models import Agency, Location, Study, Survey
+from leaflet.forms.widgets import LeafletWidget
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Submit
+
+from pldp.models import Agency, Location, Study, StudyArea
 from .models import SurveyFormEntry, SurveyChart
 
 
 class CreateAgencyForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(CreateAgencyForm, self).__init__(*args, **kwargs)
+        create_default_helper(self)
+
     class Meta:
         model = Agency
         fields = '__all__'
 
 
 class CreateLocationForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(CreateLocationForm, self).__init__(*args, **kwargs)
+        create_default_helper(self)
+
     class Meta:
         model = Location
         fields = '__all__'
 
 
+class StudyAreaCreateForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(StudyAreaCreateForm, self).__init__(*args, **kwargs)
+        create_default_helper(self)
+
+    class Meta:
+        model = StudyArea
+        fields = '__all__'
+
+        leaflet_widget_attrs = {
+            'map_height': '400px',
+            'map_width': '100%',
+            'display_raw': 'true',
```

Comment:
Does this option toggle the `Geojson` box? If so, what do you think about hiding it for the time being? It's a cool feature but it sort of overwhelms the page and I think it'll take a bit of work to get right. My instinct is to see whether this is something UCD wants and then invest the time in it if it is.

Reply:
Sure! If it comes up we can let UCD know this is an option.

Reply:
@beamalsky Looks like this straggler got lost in the shuffle, but once `dislpay_raw` is set to `false` and the GeoJSON box doesn't show I consider this ready to merge!

Reply:
Ah thanks for the catch!